src/ActionsImporter/Commands/Common.cs

```diff
@@ -19,6 +19,12 @@ public static class Common
         IsHidden = true
     };
 
+    public static readonly Option<bool> NoHostNetwork = new("--no-host-network")
+    {
+        Description = "Disable using host network.",
```

Comment:
I wonder if we should elaborate on this slightly? 

Maybe:
```suggestion
        Description = "Disable using host network and instead use the Docker network.",
```

I'm not 100% sure if I like that, but adding more detail may be useful?

Reply:
```suggestion
        Description = "Use docker's default bridge network instead of the host machine's network.",
```

I like adding some more context here but not sure what wording is best 💭